initfiles/bin/init_thorslave.in

```diff
@@ -74,9 +74,13 @@ start_slaves()
     # is sudo required for checking status ?
     ${cmd_prefix} ${INIT_PATH}/dafilesrv status > /dev/null
```

Comment:
If dafilesrv is not running this command will take 5 seconds to return a 255 value.
Perhaps we could check the process is running or its sentinel file exists before doing this or instead of doing this ?

Reply:
Fixed by removing the dafilesrv status call and using the killall in it's place.  Killall will be checking just for the existence of dafilesrv which is what we want.  It won't attempt a start if another init_dafilesrv is trying to start a healthy dafilesrv process.  Should eliminate the possible 5 seconds of lag time.